tests/system/action/projector/test_update.py

```diff
@@ -0,0 +1,58 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorUpdate(BaseActionTestCase):
+    def test_update_correct(self) -> None:
+        self.set_models({"projector/111": {"name": "name_srtgb123"}})
+        response = self.request(
+            "projector.update",
+            {
+                "id": 111,
+                "name": "name_Xcdfgee",
+                "width": 100,
+                "aspect_ratio_numerator": 3,
+                "aspect_ratio_denominator": 4,
+                "color": "ffffff",
```

Comment:
All colors have to be prefixed with a `#`. Please also add tests that check that non-compliant color strings are not accepted (the implementation for that is in #482).

Reply:
Add the prefix. If #482  is merged, I can add a non-correct color string.